frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.test.tsx

```
@@ -47,35 +41,27 @@
   beforeEach(() => {
-    (findLayoutsContainingDuplicateComponents as jest.Mock).mockReturnValue([]);
+    jest.clearAllMocks();
+    jest.restoreAllMocks();
   });
 
-  it('render heading with layout page name when layout is selected', () => {
-    const newSelectedPage = 'newSelectedPage';
-    renderPageConfigPanel(newSelectedPage);
+  it('render heading with layout page name when layout is selected', async () => {
+    const newSelectedPage = layout1NameMock;
+    await act(async () => {
```

ESLint

[error] Avoid wrapping Testing Library util calls in `act`

(testing-library/no-unnecessary-act)